### ISSUE TITLE
`2022.4.22` supports Python >= 3.8, not >= 3.7

### ISSUE THREAD

**Issue by maximlt** [conda-forge/tifffile-feedstock#93]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

hvPlot's CI has started to fail after the latest release of tifffile, the minimum Python version supported of `tifffile` hasn't been bumped to `>=3.8` in the recipe.

### Installed packages

```shell
conda list
```

### Environment info

```shell
conda info
```

**Comment by ericpre** [comment]
Actually, it seems to be the case since version 2022.2.2: https://github.com/cgohlke/tifffile/commit/aa31ae641bc2484045333f3383e45ceede617c70 and it has been missed!

**Comment by maximlt** [comment]
Oh ok! A patch to conda-forge repo data will have to be submitted.

**Comment by ericpre** [comment]
Yes, this is correct, along with a patch to this feedstock! Are you interested in making these patches?
